code/modules/mob/living/simple_animal/abnormality/teth/ppodae.dm

```diff
@@ -139,12 +139,17 @@
 						continue
 					area_of_effect += U
 		else
-			return
+			for(var/turf/T in view(1, src))
+				if (T.density)
+					break
+				if (T in area_of_effect)
+					continue
+				area_of_effect += T
```

Comment:
You might want to use |= here, so it doesn't add same turfs twice.

Reply:
I'm actually unsure where you meant by this, sorry
Since as I understand it, it needs += in the for loop

Reply:
Done